### PR TITLE
EDMA[3523] Added Modal and SnackBar to home

### PR DESCRIPTION
**Pull request by robertwollny-pnmac**

### CODE REVIEW

**Review by sfowles — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -46,7 +48,7 @@
     ]
   },
   "devDependencies": {
-    "@edma/design-tokens": "^4.2.5",
+    "@edma/design-tokens": "^4.3.0",
```
Comment (sfowles) [inline package.json:50]: Double check the Design Token docs to make sure we didn't accidentally break anything by updating this.

File: src/components/Reuseable-Components/Presentational/Modal/Modal.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```
Comment (sfowles) [inline src/components/Reuseable-Components/Presentational/Modal/Modal.js]: This looks good, but if we externalize the React components into a separate repo, and then import them like we normally would into an app, we can then simplify the dir structure to something like `src/components/React/Modal/Modal.js`

File: src/components/Reuseable-Components/Scenes/CraftModal.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
+import React, { useState } from 'react';
```
Comment (sfowles) [inline src/components/Reuseable-Components/Scenes/CraftModal.js]: Looks good as well, but we probably need to make a new repo for React components and then include this in there. The Home docs are really meant to just be the place we document our components, if that makes sense.

File: src/components/Reuseable-Components/Scenes/CraftSnackbar.js [hunk 4]
```diff
@@ -0,0 +1,238 @@
+import React, { useState } from 'react';
```
Comment (sfowles) [inline src/components/Reuseable-Components/Scenes/CraftSnackbar.js]: See above

File: src/components/Reuseable-Components/helpers/variable.js [hunk 5]
```diff
@@ -0,0 +1,226 @@
+export const exampleBasicRender = 'Basic Modal Body';
```
Comment (sfowles) [inline src/components/Reuseable-Components/helpers/variable.js]: See above

File: src/components/Reuseable-Components/Presentational/Snackbar/Snackbar.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```
Comment (sfowles) [inline src/components/Reuseable-Components/Presentational/Snackbar/Snackbar.js]: See above

File: package.json (hunk 1 above)
Comment (robertwollny-pnmac) [reply package.json:50]: 4.3.0 seems to be the correct version based on the tokens repo

**Pull request merged**